pkg/ConjuntoAnalisis.go

```diff
@@ -0,0 +1,5 @@
+package pkg
+
+type ConjuntoAnalisis struct{
```

Comment:
```suggestion
type AnalisisIndividual struct{
```
Le cambiaría el nombre a algo de este estilo para que sea más facil entender qué almacena cada clase